docs/includes/es5-support.md

```diff
@@ -0,0 +1,4 @@
+For some versions of Office and Windows, the JavaScript engine in which add-ins run is provided by Internet Explorer. (For details of which versions, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).) The Internet Explorer engine does not support versions of JavaScript later than ES5. This means that without special handling the JavaScript files that your add-in serves cannot use syntax, types, or methods that were added to the language after ES5. This does not mean that you must *write* in ES5 syntax. You have two other options:
```

Comment:
I'm not sure this follows MSFT style guidelines, but I'd find this to be more readable if the link to the "Browsers used" article was in the first sentence:
"For [some versions of Office and Windows](../concepts/browsers-used-by-office-web-add-ins.md), the JavaScript engine in which add-ins run is provided by Internet Explorer."